src/main/java/com/elytradev/concrete/block/ItemDropBehaviour.java

```diff
@@ -5,7 +5,7 @@
 import java.util.Random;
 
 /**
- * An interface used to describe the behaviour for item drops for an {@link ConcreteBlock}.
+ * A interface used to describe the behaviour for item drops for an {@link ConcreteBlock}.
```

Comment:
An was correct here, I was referring to `an ConcreteBlock` - this was correct for Unnamed, as it starts with a vowel sound, but Concrete does not.